libsel4/sel4_arch_include/x86_64/interfaces/sel4arch.xml

```diff
@@ -28,7 +28,27 @@
         <member name="fs_base"/>
         <member name="gs_base"/>
     </struct>
-
+    <struct name="seL4_VCPUContext">
+        <member name="rax"/>
+        <member name="rbx"/>
+        <member name="rcx"/>
+        <member name="rdx"/>
+        <member name="rsi"/>
+        <member name="rdi"/>
+        <member name="rbp"/>
```

Comment:
@abrandnewusername , i tried doing this too, but unfortunately, there's no easy way without more changes to the tooling to conditionally select the fields in the struct definition based on the `CONFIG_X86_64_VTX_64BIT_GUESTS` configuration. This is what's now causing build errors for existing configurations. Properly supporting this would require adding support for `<condition>`'s within struct declarations, however I felt that it would be acceptable to keep the e' prefixed names for the initial PR and it could be optionally fixed up later (otherwise it may take quite a while for this to get merged)

Reply:
Fair point. I will make a GitHub issue for this.